README.md

```diff
@@ -1,2 +1,350 @@
+
+---
+
+![run lsa](/images/run_ls.png)
+
+**`/run`** is the run-time variable data. It is a temporary location for system processes to store their own temporary data. When you boot your system, this folder is probably empty and starts to get populated as your system operates.  It is relatively a new directory. Like **`/proc`**, it stores run-time volatile data.
+
+---
+
+This one is pretty huge too...
+
+**`/sbin`** has the same duty as the **`/bin`** directory but it is for the superuser - 's' in the beginning for that reason -. You use those programs with the `sudo` command which **temporarily** renders you "super", and gives you powers of the superuser.
+As you may infer, **`/sbin`** contains mutators, and programs that change something within the system such as installing, deleting, or formatting/changing a component or package. The essential binaries to boot, restore, recover, and repair the system are here too. For example, updating passwords for users `chgpasswd [group_name:password]` resides here. 
+
+![chgpasswd place](/images/chgpasswd_place.png)
+
+---
+
+![lsa snap ](/images/lsa_snap.png)
+
+**`/snap`** is for storing your favorite Snapchat stories. I am kidding. Let us read what the `README` says within the **`/snap`** directory. 
+
+![snap readme](/images/snap_readme.png)
+
+Let me refresh your memory: `apt` was a cli handling installation or removing the packages. It is a packaging tool. 'snap' - not the directory, the program - is similar, it is a packaging and deployment system. Snaps are a secure and scalable way to embed applications on Linux devices. A snap is an application **containerized** with all its dependencies. A snap can be installed using a single command on any device running Linux. Applications run fully isolated in their own sandbox, thus minimizing security risks. Thought, there was an incident, where in a snap, the developer put cryptocurrency mining software and all hell broke loose...
+
+Those packages are also called snaps. Snaps are self-contained applications running in a sandbox with mediated access to the host system. Self-contained means they consist of a single, installable bundle containing the application and a copy of the run-time environment needed to run the application. When the application is installed, it behaves in the same way as any native application. The file format for snap is a single compressed filesystem with the extension `.snap`. This filesystem contains the application, libraries it depends on, and declarative metadata. This metadata is interpreted by snapd to set up an appropriately shaped secure sandbox for that application. After installation, the snap is mounted - this is mediated access we talked about above - by the host operating system and decompressed on the fly when the files are used. Although this has the advantage that snaps use less disk space, it also means some large applications start more slowly.
```

Comment:
> Those packages are also called snaps

did not understand what does this `those packages` actually refer to :(

Reply:
This part was a bummer, I am glad that it didn't slip. When I was writing I dumped everything, and pasted it together. I was not sure what to do here. Snaps are zip files. Their data is zipped. On the fly, snapd unzips them. They also share common dependencies. I wrote they use less space, but I am not sure if this holds because what do we compare them to? Common dependencies should be shared among anything practically in the OS as long as I am aware. SO I thought I deleted this part but it turns out it slipped in. What do you think?

Reply:
Yes, I think deleting that part and not confusing the reader is a better choice. This is not about snaps in the end. It was a detail 👍